### PR TITLE
config changes for arbitrum

### PR DESCRIPTION
**Pull request by Arham-ifran**

### CODE REVIEW

File: scripts/helpers/etherscan.ts [hunk 1]
```diff
@@ -8,6 +8,10 @@ export default (type: EtherscaneType, hash: string): string => {
     return `https://etherscan.io/${type}/${hash}`
   } else if (networkName === 'goerli') {
     return `https://goerli.etherscan.io/${type}/${hash}`
+  } else if (networkName === 'arbitrumsepolia') {
+    return `https://sepolia.arbiscan.io/${type}/${hash}`
+  } else if (networkName === 'arbitrumOne') {
+    return `https://arbiscan.io/${type}/${hash}`
```
Comment (calvinh8) [inline scripts/helpers/etherscan.ts:14]: Use switch case

File: scripts/deploy-token.ts [hunk 2]
```diff
@@ -18,16 +18,10 @@ async function main() {
   const STMXContract = await ethers.getContractFactory('STMX', owner)
 
   // 18 decimal places, convert to full units
-  const initialSupply = BigNumber.from(values.INITIAL_SUPPLY).mul(BigNumber.from(10).pow(18))
-  const args = [values.NAME, values.SYMBOL, initialSupply, owner.address]
-  const token = await upgrades.deployProxy(STMXContract, args) as STMX
-
-  console.info(`Check tx here: ${getEtherscanUri('tx', token.deployTransaction.hash)}`)
-
-  await token.deployed()
```
Comment (calvinh8) [inline scripts/deploy-token.ts:27]: How do you deploy if this is removed?

File: scripts/deploy-staking.ts [hunk 3]
```diff
@@ -17,12 +17,7 @@ async function main() {
   const StakingContract = await ethers.getContractFactory('Staking', owner)
   const staking: Staking = await StakingContract.deploy(values.NEW_TOKEN, values.TREASURY)
 
-  console.info(`Check tx here: ${getEtherscanUri('tx', staking.deployTransaction.hash)}`)
-
-  await staking.deployed()
```
Comment (calvinh8) [inline scripts/deploy-staking.ts:22]: How do you deploy if this is removed?

**Review by calvinh8 — CHANGES_REQUESTED** [review]

File: scripts/helpers/etherscan.ts (hunk 1 above)
Comment (Arham-ifran) [reply scripts/helpers/etherscan.ts]: Done. Please check

File: scripts/deploy-token.ts (hunk 2 above)
Comment (Arham-ifran) [reply scripts/deploy-token.ts:27]: .deployed() is only used to wait for the deployment. The contract is actually deployed by calling the method .deploy(), which is already there. 

Also, this .deployed() method is not available in the new versions that why I removed it. Now I have added the alternate which is .waitForDeployment()

File: scripts/deploy-staking.ts (hunk 3 above)
Comment (Arham-ifran) [reply scripts/deploy-staking.ts:22]: .deployed() is only used to wait for the deployment. The contract is actually deployed by calling the method .deploy(), which is already there. 

Also, this .deployed() method is not available in the new versions that why I removed it. Now I have added the alternate which is .waitForDeployment()

**Review by calvinh8 — APPROVED** [review]

**Pull request merged**